### ISSUE TITLE
Synchronization with the NISO RP

### ISSUE THREAD

**Issue by Klortho** [CrossRef/niso-ali#7]
@kjw, This is a follow-on to #1.  You referenced the issue of synchronization of this repository with the [NISO Recommended Practice](http://www.niso.org/apps/group_public/download.php/14226/rp-22-2015_ALI.pdf), and I think that's really where the heart of the problem lies.  The RP document itself does not in any way reference this repository, from what I can tell. It is referenced on the [ALI Schemas page](http://www.niso.org/schemas/ali/1.0/). But there's no indication of what is "canonical".

Moreover, the versioning is a bit messed up, I think.  There's no clear idea of what version "1.0" means.

Could I suggest:
1. You create a tag of this repository with "1.0", for whichever commit matches the code in the specification
2. Change the names of the schema files in this repository to match the deployed names:
   - niso-ali-1.0.xsd -> ali.xsd
   - niso-ali-1.0.json -> jsonld.json
3. ~~(In fact, if it's not too late, it would be nice to change the latter to ali.jsonld, for symmetry)~~
4. NISO site should have a "current" version and a "1.0" version (and a new version each time they get released):
   - 1.0 (same as existing, but "downdated" to match what's in the RP):
     - http://www.niso.org/schemas/ali/1.0/ali.xsd
     - http://www.niso.org/schemas/ali/1.0/jsonld.json
   - current (should track master here):
     - http://www.niso.org/schemas/ali/ali.xsd
     - http://www.niso.org/schemas/ali/jsonld.json
5. The RP should be amended to reference this repository, with this versioning scheme described
6. The [NISO ALI Schemas page](http://www.niso.org/schemas/ali/1.0/) should likewise be amended.

How do you communicate with the NISO folks?  I have had very little luck getting answers.  I'd be willing to help with some of this if you agree to it, and you need the help.

**Comment by kjw** [comment]
Agree with all your comments here and will act upon the points I'm able to (anything relating to this repo.)

I've been communicating with the NISO group through CrossRef's representative on the group, Ed Pentz. In fact that's how this repo was started - a draft of the NISO recommendation had examples and schemas that did not match up. I attempted to produce here something coherent, and the last I heard was that these examples and schemas would be linked to and documents updated accordingly. Though for an accurate and up-to-date summary please speak to the NISO group directly. I get the impression I don't have a full and accurate picture of what was decided.

I just heard by way of other people that the NISO group is now disbanded and potentially would not want to make any changes so soon after their 'release'. Don't take that as official word though - it's about third or forth hand information. Communication should go direct to them.

**Comment by kjw** [comment]
OK looking into this more. The schemas (both XML and JSON-LD) available via the links you provide above match the schemas currently in this repository.

What has not been updated is the verbatim copy of the XML schema in the RP document in appendix A, and the JSON-LD examples also in appendix A that utilize a boolean `free_to_read` - something the schema now forbids.

I can mark a relevant commit with `1.0` tag but it will not get around the problem that the schemas on the NISO site and here agree with each other, while the XML schema and JSON-LD examples in the RP document do not agree with the schema files here and on the NISO site.

**Comment by ckoscher** [comment]
I believe the RP needs too be updated. The RP was issued in January but 
the schema files are more recent (February). It appears that changed 
schema files were posted without going back to the RP.

I've emailed Todd (Niso) about these discrepancies but have not heard 
back yet.

Chuck

**Comment by Klortho** [comment]
In continuing pursuit of getting this RP and repository in sync, I submitted a couple of pull request.

Still needs to be done:
- Tag commit b6421bac with "1.0"
- Deploy those versions of the ali.xsd and jsonld.json to the niso site, under http://www.niso.org/schemas/ali/1.0/
- Deploy the latest master versions of these to http://www.niso.org/schemas/ali/
- Editing and publishing an updated version of the RP
- Updating the [NISO ALI schemas page](http://www.niso.org/schemas/ali/1.0/)

**Comment by kjw** [comment]
@Klortho - I'm waiting to hear why the RP was left out of sync with the changes made to the schema, namely removing the option of a boolean free-to-read in favour of a free-to-read period with a start date equal to publication date and without an end date. They chose to adopt this schema, but did not change the RP. Did they misunderstand what they were adopting, or not realise that it would require a change to the RP?

I'd like to know their intention before making changes.

On JeffSpies/SHARE-Schema#10 - two problems.

1) The discussion suggests the need to represent 'isOpenAccess' and 'isPublic' using a boolean version of free-to-read. Free to read, though, does not indicate open access, it indicates public availability, without payment, for reading. Nothing else. Ideas about open access should be expressed through license selection.

2) Still don't get why it is difficult to represent 'isPublic = true' as a free-to-read period starting with publication date and never ending.

**Comment by Klortho** [comment]
> 2) Still don't get why it is difficult to represent 'isPublic = true' as a free-to-read period starting with publication date and never ending.

It's just unnecessary added friction.

**Comment by Klortho** [comment]
Regarding your problem #1 above, I don't interpret it quite the same way.  It seems to me that @erinspace is saying that they just want to implement the logic, "IF isOpenAccess OR isPublic THEN free-to-read".  Not that she is wanting to _replace_ either of those two flags with free-to-read.

**Comment by kjw** [comment]
@Klortho Yes agreed. isOpenAccess implies free-to-read is probably a reasonable assumption. But, if the logic is just that, without any additional translation of isOpenAccess, then important information is being lost.

I worry about the confusion of open access and free-to-read. One of the reasons I do not like the free-to-read tag.
